src/main/java/uk/gov/register/core/EntryLog.java

```diff
@@ -73,15 +73,21 @@ public RegisterProof getRegisterProof(int entryNo) {
     public EntryProof getEntryProof(int entryNumber, int totalEntries) {
         List<String> auditProof = backingStoreDriver.withVerifiableLog(verifiableLog ->
                 verifiableLog.auditProof(entryNumber, totalEntries)
-                        .stream().map(this::bytesToString).collect(Collectors.toList()));
+                        .stream()
+                        .map(this::bytesToString)
+                        .map(p -> HashingAlgorithm.SHA256.toString() + ":" + p)
+                        .collect(Collectors.toList()));
 
         return new EntryProof(Integer.toString(entryNumber), auditProof);
     }
 
     public ConsistencyProof getConsistencyProof(int totalEntries1, int totalEntries2) {
         List<String> consistencyProof = backingStoreDriver.withVerifiableLog(verifiableLog ->
                 verifiableLog.consistencyProof(totalEntries1, totalEntries2))
-                .stream().map(this::bytesToString).collect(Collectors.toList());
+                .stream()
+                .map(this::bytesToString)
+                .map(p -> HashingAlgorithm.SHA256.toString() + ":" + p)
```

Comment:
Same as above. `ConsistencyProof.consistencyNodes` should be changed to be of type `List<HashValue>` and that will mean you will not need to repeat string concatenation here.